fly/eventstream/render.go

```diff
@@ -39,7 +39,7 @@ func Render(dst io.Writer, src eventstream.EventStream, options RenderOptions) i
 
 		case event.SelectedWorker:
 			dstImpl.SetTimestamp(e.Time)
-			fmt.Fprintf(dstImpl, "\x1b[1mselected worker: %s\n", e.WorkerName)
+			fmt.Fprintf(dstImpl, "\x1b[1mselected worker:\x1b[0m %s\n", e.WorkerName)
```

Comment:
Looks like this broke the unit test actually: https://ci.concourse-ci.org/builds/2054579#L5f1da44d:550:553

Reply:
My bad. Forgot to update unit test, and after pushed the commit, didn't get a chance to come back to check build result. Just fixed.